text/0092-replay-issue-creation.md

```diff
@@ -17,43 +17,66 @@ These issues can only be detected on the SDK. The Replay back-end will never hav
 
 # Options Considered
 
-### Option 1: SDK Generates Issues Through a Generic Interface
+### Option 1: SDK Creates Issues Through the Error Interface
```

Comment:
How about just:
```suggestion
### Option 1: SDK Creates Issues Through the SDK
```

Reply:
I think we should change it to `captureException` to be clear the mechanism we're using.  "SDK ... Through the SDK" is a bit circular.